py/desisim/scripts/quickbrick.py

```diff
@@ -120,9 +120,18 @@ def main(args):
     log.debug('Using OBJTYPE {}'.format(objtype))
     if objtype == 'BGS' or objtype == 'MWS' or objtype == 'BRIGHT_MIX':
         qsim.observation.exposure_time = desiparams['exptime_bright'] * u.s
-        qsim.atmosphere.moon.moon_zenith = args.moon_zenith * u.deg
-        qsim.atmosphere.moon.separation_angle = args.moon_angle * u.deg
-        qsim.atmosphere.moon.moon_phase = args.moon_phase
+        if args.moon-phase is None:
+            qsim.atmosphere.moon.moon_phase = 0.7
+        else:
+            qsim.atmosphere.moon.moon_phase = args.moon-phase
```

Comment:
I can certainly write a test for this, but I'm not exactly sure what that test would entail. I changed the keywords in quickbrick to mimic the parameters of specsim, but that seems to be more of a change with respect to convention, rather than being something intrinsically prone to errors.